src/funcConfig/local.settings.ts

```diff
@@ -21,53 +20,25 @@ export interface ILocalSettingsJson {
 
 export const azureWebJobsStorageKey: string = 'AzureWebJobsStorage';
 
-export async function validateAzureWebJobsStorage(actionContext: IActionContext, localSettingsPath: string): Promise<void> {
+export async function getAzureWebJobsStorage(projectPath: string): Promise<string | undefined> {
     // func cli uses environment variable if it's defined on the machine, so no need to prompt
     if (process.env[azureWebJobsStorageKey]) {
-        return;
-    }
-
-    const settings: ILocalSettingsJson = await getLocalSettingsJson(localSettingsPath);
-    if (settings.Values && settings.Values[azureWebJobsStorageKey]) {
-        return;
+        return process.env[azureWebJobsStorageKey];
     }
 
-    const message: string = localize('azFunc.AzureWebJobsStorageWarning', 'All non-HTTP triggers require AzureWebJobsStorage to be set in \'{0}\' for local debugging.', localSettingsFileName);
-    const selectStorageAccount: vscode.MessageItem = { title: localize('azFunc.SelectStorageAccount', 'Select Storage Account') };
-    const result: vscode.MessageItem = await ext.ui.showWarningMessage(message, selectStorageAccount, DialogResponses.skipForNow);
-    if (result === selectStorageAccount) {
-        const resourceResult: azUtil.IResourceResult = await azUtil.promptForStorageAccount(
-            actionContext,
-            {
-                kind: [
-                    StorageAccountKind.BlobStorage
-                ],
-                performance: [
-                    StorageAccountPerformance.Premium
-                ],
-                replication: [
-                    StorageAccountReplication.ZRS
-                ],
-                learnMoreLink: 'https://aka.ms/Cfqnrc'
-            }
-        );
-
-        // tslint:disable-next-line:strict-boolean-expressions
-        settings.Values = settings.Values || {};
-        settings.Values[azureWebJobsStorageKey] = resourceResult.connectionString;
-        await fsUtil.writeFormattedJson(localSettingsPath, settings);
-    }
+    const settings: ILocalSettingsJson = await getLocalSettingsJson(path.join(projectPath, localSettingsFileName));
+    return settings.Values && settings.Values[azureWebJobsStorageKey];
```

Comment:
Is this returning a string or a boolean?  I thought `return settings.Values && settings.Values[azureWebJobsStorageKey];` would return `true` or `false.`

Reply:
Nah that returns a string. The build passed - do you not believe typescript? haha

Reply:
Haha, fair enough.  I tried it out to validate too-- that's super weird though.